src/user.c

```diff
@@ -40,6 +40,16 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 extern int eos;
 
+/** Lookup an account from a Nick 
+ * @param   nick    pointer to a Nick
+ * @returns an User struct
+ * @note    This is an helper function to ease the migration to an account based 
+ *          paradigm.
+ */
+User *find_account(Nick *nick) {
```

Comment:
Nit: the de-facto coding style uses a newline before the opening bracket for functions. For consistency, could you do the same here?
Nit: a Nick* variable is usually named "nptr" rather than "nick". Could you do the same here?

Reply:
Fixed